src/systems/light.js

```diff
@@ -73,5 +98,17 @@ module.exports.System = registerSystem('light', {
     sceneEl.appendChild(directionalLight);
 
     this.defaultLights = true;
+  },
+
+  /**
+   * Enables/disables the renderer shadow map.
+   * @param {boolean} enabled
+   */
+  setShadowMapEnabled: function (enabled) {
+    var renderer = this.sceneEl.renderer;
+    this.shadowMapEnabled = enabled;
+    if (renderer) {
+      renderer.shadowMap.enabled = enabled;
+    }
```

Comment:
I don't see this in most shadow demos/docs, is there a particular type of material this effects?